wiki/en/en-Installation-for-Linux.md

```diff
@@ -15,25 +15,35 @@ Make sure you read the [Getting Started](Getting-Started) page.
 Although we don't provide official packages for every Linux distribution, you might find Jamulus in the package manager of your distro. Have a look at this table:
 
 [![Packaging status](https://repology.org/badge/vertical-allrepos/jamulus.svg)](https://repology.org/project/jamulus/versions)
-
+P
 
 There is also list of unofficial [packages for different distributions here](https://github.com/jamulussoftware/jamulus/discussions/914).
 
 ### Flatpak
 
-Jamulus is available as a [Flatpak](https://flatpak.org/) from [Flathub here](https://flathub.org/apps/details/io.jamulus.Jamulus). You will need to setup pipewire in order to use the Jamulus flatpak.
+Jamulus is available as a [Flatpak](https://flatpak.org/) from [Flathub here](https://flathub.org/apps/details/io.jamulus.Jamulus). You will need to set up [PipeWire](https://pipewire.org/) in order to use the Jamulus flatpak.
 
 ### Debian and Ubuntu
 You can install Jamulus from [buster-backports](https://packages.debian.org/de/buster-backports/jamulus) or - if you're on amd64 **Debian**/**Ubuntu** - you may try the compiled .deb packages from GitHub Actions:
 
 1. Download the [latest .deb file]({{ site.download_root_link }}{{ site.download_file_names.deb-gui }})
-1. Update apt: `sudo apt-get update`
+1. Update apt: `sudo apt update`
 1. Install the package: `sudo apt install /path/to/{{ site.download_file_names.deb-gui }}`.
-1. Since Jamulus needs the JACK server, you have to install it too. We recommend to use `QjackCtl` to configure JACK. You can install it via `sudo apt-get install qjackctl`
+1. Since Jamulus needs the JACK server, you have to install it too. We recommend to use `QjackCtl` to configure JACK. You can install it via `sudo apt install qjackctl`
 
 Afterwards you should look at the "[Set up your sound card](#set-up-your-sound-card)" step on this page.
 
-**Note:** Jamulus (with GUI) is also [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt-get install jamulus`.
+**Note:** Jamulus (with GUI) is also [included in Debian Bullseye (testing)](https://packages.debian.org/bullseye/jamulus) thanks to [mirabilos](https://github.com/mirabilos). If you already use Bullseye, just install Jamulus via `sudo apt install jamulus`.
+
+### PipeWire
```

Comment:
I think the section of setting up pipewire should go into the community knowledge base (see the _posts dir).

Reply:
Thank you for your comment. I intentionally left out how to set it up. Do you mean my comment about how PipeWire worked better on my system than the jackd?

Reply:
I'd totally remove the pipewire reference here and give a link to another page with more explanation. Keep this even shorter (or remove it completely and give a link to the Knowledge Base).